src/core/qgscoordinatereferencesystem.cpp

```diff
@@ -1072,7 +1072,11 @@ QgsRectangle QgsCoordinateReferenceSystem::bounds() const
       double north = statement.columnAsDouble( 1 );
       double east = statement.columnAsDouble( 2 );
       double south = statement.columnAsDouble( 3 );
-      rect = QgsRectangle( west, north, east, south );
+
+      rect.setXMinimum( west );
+      rect.setYMinimum( north );
+      rect.setXMaximum( east );
+      rect.setYMaximum( south );
     }
```

Comment:
Can you add a unit test for one of these projections? I'm concerned someone will try to optimise this in future.